packages/theme/src/cli/utilities/theme-fs.ts

```diff
@@ -83,7 +90,12 @@ export async function readThemeFile(root: string, path: Key) {
     return
   }
 
-  return readFile(absolutePath, options)
+  const value = await readFile(absolutePath, options)
```

Comment:
After typing this up, I ended up changing implementation given this:

> Returning a Buffer here is better for clarity for downstream consumers, as they will clearly know whether the data returned is related to an image by inspecting the data itself, rather than having to lookup the mimetype of the file as done here